README.md

```diff
@@ -1,34 +1,73 @@
-# bookay
 
-[![Build Status](https://github.com/jaynetics/bookay/workflows/tests/badge.svg)](https://github.com/jaynetics/bookay/actions)
+<br>
 
-bookay is a set of free, open-source tools for self-hosting your bookmarks.
+<div align = center>
 
-[
-  ![Screenshot thumbnail showing the webapp and browser plugin](
-    https://user-images.githubusercontent.com/10758879/113178657-1528d000-924f-11eb-9a02-2c0e4e504074.png
-  )
-](https://user-images.githubusercontent.com/10758879/113178042-77350580-924e-11eb-820f-298da2a2631d.png)
+[![Badge Build]][Actions]   [![Badge Heroku]][Heroku]
```

Comment:
i'm not a fan of the extra level of indirection introduced by these link references. i'd rather have my README easy to edit. the indirection would only make sense if the same link was repeated often, IMO.

Reply:
No.

> the indirection would only make sense if the same link was repeated often, IMO.

Just no.

The sole purpose of link references isn't to be used for multiple similar links, it's to decouple the content from the location it's pointing to.

> easy to edit

Easy to edit? You call having to edit a link in a reference difficult compared to editing it in the flow of text? Seems quite the opposite considering I added a dedicated section for that, you now don't have to select two varying point in the flow of text but rather just a static start.